packages/workflow/src/ExecutionStatus.ts

```diff
@@ -1,13 +1,13 @@
 export const ExecutionStatusList = [
-	'canceled' as const,
```

Comment:
why write something once when it can be repeated 9 times 😅.
Thanks for fixing this.